packages/ics721/src/testing/ibc_tests.rs

```diff
@@ -651,3 +653,183 @@ fn test_no_receive_when_paused() {
         .unwrap()
         .starts_with("contract is paused pending governance intervention"))
 }
+
+#[test]
+fn test_ibc_packet_receive_callback() {
```

Comment:
Just realized, we dont have any unit tests for `ibc_packet_ack()` and `ibc_packet_timeout()`. Imo not important, since tx is atomic and nothing can go wrong here. At least we should create an issue - for covering this as well.

Reply:
I think e2e test cover this in length and good enough.